server/routes/requests.js

```diff
@@ -132,6 +132,7 @@ router.patch('/updateRequest/:id/:timeOffset', passport.authenticate('jwt', {ses
             const dbData = await requestSchema.find({mentor: alumniId, status: status}).populate('student')
             for (let request of dbData) {
                 request.time = await timezoneHelpers.applyTimezone(request.time, timeOffset)
+                request.privateFeedback = undefined;
```

Comment:
This removes any evidence of privateFeedback from returned documents

Reply:
Perfect! And we're not losing any of this data on fetches, so that's great